config/v1/techpreview.infrastructure.testsuite.yaml

```diff
@@ -12,3 +12,137 @@ tests:
       apiVersion: config.openshift.io/v1
       kind: Infrastructure
       spec: {}
+  onUpdate:
```

Comment:
Can you add a test case that checks, from an `initial` with  the `UserManaged` case, what happens if I try to remove the `loadBalancer` and it's subkeys from the object? I suspect right now that will be accepted and we will need an additional validation to make sure the field cannot be unset (will need the same for the type within the load balancer as well)

Reply:
I've addressed the comment on the validation. However I think the testsuite is ok, unless I missed something?